mycroft/client/enclosure/eyes.py

```diff
@@ -67,6 +69,24 @@ def color(self, event=None):
         color = (r * 65536) + (g * 256) + b
         self.writer.write("eyes.color=" + str(color))
 
+    def set_pixel(self, event=None):
+        idx = 0
+        r, g, b = 255, 255, 255
+        if event and event.data:
+            idx = int(event.data.get("idx", idx))
+            r = int(event.data.get("r", r))
+            g = int(event.data.get("g", g))
+            b = int(event.data.get("b", b))
+        color = (r * 65536) + (g * 256) + b
+        self.writer.write("eyes.set=" + str(idx) + "," + str(color))
+
+    def fill(self, event=None):
+        amount = 0
+        if event and event.data:
+            percent = int(event.data.get("percentage", 0))
+            amount = round(23.0 * percent / 100.0)
```

Comment:
This doesn't seem to cause troubles, but I think we should cast to int since in python 2, `round` still returns a float.